helm/botkube/values.yaml

```diff
@@ -46,6 +46,8 @@ kubeconfig:
 
 # set one of the log levels- info, warn, debug, error, fatal, panic
 logLevel: info
+# set to true to enable ANSI color in logging
+logForceColors: false
```

Comment:
Please also update Readme (currently it is done manually): https://github.com/infracloudio/botkube/blob/develop/helm/botkube/README.md

Reply:
Sure